artemis-boot/src/main/java/org/apache/activemq/artemis/boot/Artemis.java

```diff
@@ -46,7 +46,11 @@ public static void main(String[] args) throws Throwable {
 
       String instance = System.getProperty("artemis.instance");
       File fileInstance = instance != null ? new File(instance) : null;
-      execute(fileHome, fileInstance, args);
+
+      Object result = execute(fileHome, fileInstance, args);
+      if (result instanceof Exception) {
```

Comment:
this will silently fail the CLI. shouldn't you System.out about the exception here?

In what situation this will return Exception? I coudln't see it

Reply:
Just saw it.. you should definitely System.out

```
System.out.println("Execution error: result.getMessage()")
```


notice we use System.out on the CLI as it's an interface with the user.

Reply:
The exception messages are already printed in the execute method.

Reply:
Ok, can you add some commented doc stating that please?